_P208_Nokia_LCD_5110.ino

```diff
@@ -0,0 +1,290 @@
+        byte choice7 = Settings.TaskDevicePluginConfig[event->TaskIndex][5];
+        int optionValues5[3] = { 1,2,3 };
+        String options5[3] = { F("normal"), F("large"), F("x-large") };
+        addFormSelector(F("Char.size line-1"), F("plugin_208_charsize_line_1"), 3, options5, optionValues5, choice5);
+        addFormSelector(F("Char.size line-2"), F("plugin_208_charsize_line_2"), 3, options5, optionValues5, choice6);
+        addFormSelector(F("Char.size line-3"), F("plugin_208_charsize_line_3"), 3, options5, optionValues5, choice7);
+
+        char deviceTemplate [lcd_lines][digits_per_line];
+        LoadCustomTaskSettings(event->TaskIndex, (byte*)&deviceTemplate, sizeof(deviceTemplate));
+        for (byte varNr = 0; varNr < lcd_lines; varNr++)
+        {
+          addFormTextBox(String(F("Line ")) + (varNr + 1), String(F("Plugin_208_template")) + (varNr + 1), deviceTemplate[varNr], 80);
+        }
+        success = true;
+        break;
+      }
+
+    case PLUGIN_WEBFORM_SAVE:{
+        Settings.TaskDevicePluginConfig[event->TaskIndex][0] = getFormItemInt(F("plugin_208_backlight"));
+        Settings.TaskDevicePluginConfig[event->TaskIndex][1] = getFormItemInt(F("plugin_208_contrast"));
+        Settings.TaskDevicePluginConfig[event->TaskIndex][2] = getFormItemInt(F("plugin_208_rotation"));
+        Settings.TaskDevicePluginConfig[event->TaskIndex][3] = getFormItemInt(F("plugin_208_charsize_line_1"));
+        Settings.TaskDevicePluginConfig[event->TaskIndex][4] = getFormItemInt(F("plugin_208_charsize_line_2"));
+        Settings.TaskDevicePluginConfig[event->TaskIndex][5] = getFormItemInt(F("plugin_208_charsize_line_3"));
+
+        char deviceTemplate[lcd_lines][digits_per_line];
+        for (byte varNr = 0; varNr < lcd_lines; varNr++)
+        {
+          char argc[25];
+          String arg = F("Plugin_208_template");
+          arg += varNr + 1;
+          arg.toCharArray(argc, 25);
+          String tmpString = web_server.arg(argc);
+          strncpy(deviceTemplate[varNr], tmpString.c_str(), sizeof(deviceTemplate[varNr]));
+        }
+
+      //  Settings.TaskDeviceID[event->TaskIndex] = 1; // temp fix, needs a dummy value
+
+        SaveCustomTaskSettings(event->TaskIndex, (byte*)&deviceTemplate, sizeof(deviceTemplate));
+        success = true;
+        break;
+      }
+
+    case PLUGIN_INIT:{
+        if (!lcd3)
+        {
+         // #define RST  4 //LCD_pin_1 reset
+         // #define CE   5 //LCD_pin_2 chip select
+         // #define DC  12 //LCD_pin_3 Data/Command select
+         // #define DIN 13 //LCD_pin_4 Serial data 
+         // #define CLK 14 //LCD_pin_5 Serial clock out
+         //lcd3 = new Adafruit_PCD8544(CLK, DIN, DC, CE, RST);
+
+          #if defined(ESP8266)
+              // SPI interface
+              lcd3 = new Adafruit_PCD8544(15, 2, -1); // D/C (need to be connected) GPIO2 , CS(CEN) GPIO15 , RESET not connected

```

Comment:
Can't this pin selection be made configurable?

Reply:
No. It's determed by the ESPEasy-hardware tab

Reply:
GPIO 2?

Reply:
You'r right...

Reply:
Done. pin selection are configurable in webform.

Reply:
![Knipsel](https://user-images.githubusercontent.com/79747798/111024326-80b11780-83de-11eb-993a-8b20fa3f315d.JPG)